docs/NooBaaNonContainerized/NooBaaCLI.md

```diff
@@ -0,0 +1,603 @@
+
+
+### Update Bucket
+
+The `bucket update` command is used to update an existing bucket with customizable options.
+
+#### Usage
+```sh
+noobaa-cli bucket update --name <bucket_name> [--new_name] [--owner]
+[--path][--bucket_policy][--fs_backend][--force_md5_etag][--from_file]
+```
+#### Flags -
+- `name` (Required)
+    - Type: String
+    - Description: Specifies the name of the updated bucket.
+
+- `new_name`
+    - Type: String
+    - Description: Specifies the new name of the bucket.
+
+- `owner`
+    - Type: String
+    - Description: Specifies the name of the owner of the bucket.
+
+- `path`
+    - Type: String
+    - Description: Specifies the underlying path of the bucket. Path must be an absolute path.
+
+- `bucket_policy`
+    - Type: String
+    - Description: Set the bucket policy, type is a string of valid JSON policy.
+
+- `fs_backend`
+    - Type: String
+    - Enum: none | GPFS | CEPH_FS | NFSv4
+    - Description: Specifies the file system of the bucket (default config.NSFS_NC_STORAGE_BACKEND), unset with ''.
+
+- `force_md5_etag`
+    - Type: Boolean
+    - Description: Set the bucket to force md5 ETag calculation.
+
+
+### Bucket Status
+
+The `bucket status` command is used to print the status of the bucket.
+
+#### Usage
+```sh
+noobaa-cli bucket status --name <account_name>
+```
+#### Flags -
+- `name` (Required)
+    - Type: String
+    - Description: Specifies the name of the bucket.
+
+### List buckets
+
+The `bucket list` command is used to display a list of all existing buckets.
+
+
+#### Usage
+```sh
+noobaa-cli bucket list [--wide][--name]
+```
+#### Flags -
+- `wide`
+    - Type: Boolean
+    - Description: Display additional details for each bucket.
+
+- `name`
+    - Type: String
+    - Description: Filter the list based on the provided bucket name.
+
+### Delete Bucket
+
+The `bucket delete` command is used to delete an existing bucket.
+
+
+#### Usage
+```sh
+noobaa-cli bucket delete --name <bucket_name> [--force]
+```
+#### Flags -
+- `name` (Required)
+    - Type: String
+    - Description: Specifies the name of the bucket to be deleted.
+
+- `force`
+    - Type: Boolean
+    - Description: Forcefully delete bucket if the bucket is not empty.
+
+
+## Managing Server IP white list
+
+
+### Set server IP white list
+
+The `whitelist` command is used to set white list of server IPs for S3 access, Allow access to all the IPs if list is empty.
+
+#### Usage
+```sh
+noobaa-cli whitelist --ips <ips>
+```
+#### Flags -
+- `ips` (Required)
+    - Type: String
+    - Description: Specifies the white list of server IPs for S3 access. Example - '["127.0.0.1", "192.0.10.0", "3002:0bd6:0000:0000:0000:ee00:0033:6778"]'
+
+
+## Managing Glacier
+
+TODO
+
+
+## Global Flags
+
+Global options used by the CLI to define the config directory settings. 
+
+- The usage of config_root and config_root_backend flag is not recommended for users and should be used only for dev environments. 
```

Comment:
I think it would be better to add this comment under each flag description.
Anyway, inline code for better readability: 
```suggestion
- The usage of `config_root` and `config_root_backend` flag is not recommended for users and should be used only for dev environments. 
```